ubuntu_image/common_builder.py

```diff
@@ -136,6 +140,19 @@ def load_gadget_yaml(self):
         shutil.copy(self.yaml_file_path, self.workdir)
         with open(self.yaml_file_path, 'r', encoding='utf-8') as fp:
             self.gadget = parse_yaml(fp)
+        # XXX: This is temporary
+        # Let's knowingly error out for now when someone tries to pass
+        # --cloud-init, --disable-console-conf or --disk-info when building
+        # a seeded UC20 image, as it is not supported by snapd yet.
+        if self.gadget.seeded:
+            # We should make sure to remove this once we get proper support
+            # for these in UC20.
+            if self.cloud_init:
+                raise UnsupportedFeatureError('--cloud-init')
+            elif self.disable_console_conf:
+                raise UnsupportedFeatureError('--disable-console-conf')
+            elif self.disk_info:
+                raise UnsupportedFeatureError('--disk-info')
```

Comment:
why is this not supported? imho we should just throw disk-info onto the seed partition.

Reply:
True! Guess there is no reason for this to not work the same, didn't know if this fit conceptually. Let me remove it in this case.